ADVANCED.md

```diff
@@ -284,10 +284,10 @@ export interface Props {
 
 ## `as` props (passing a component to be rendered)
 
-`ReactType` is pretty useful to cover most types that can be passed to createElement e.g.
+`ElementType` is pretty useful to cover most types that can be passed to createElement e.g.
```

Comment:
Now that I'm reading this: Shouldn't it cover **all** types?

Reply:
I'd have to double check but it *may* break on exotic components, although I do think we covered that use case too.